src/components/AddSample/AddSample.jsx

```diff
@@ -0,0 +1,378 @@
+        </div>
+        <div>
+          <Typography variant='body1'> Growth Region: </Typography>
+          <TextField
+            className={classes.inputs}
+            required
+            label='City'
+            variant='filled'
+            value={city} 
+            type="text" 
+            onChange={(event) => setCity(event.target.value)}/>
+          
+            {/* <input
+              type="text"
+              value={city}
+              onChange={(event) => setCity(event.target.value)} /> */}
+          <TextField
+            className={classes.inputs}
+            required
+            label='State'
+            variant='filled'
+            value={state} 
+            type="text" 
+            onChange={(event) => setState(event.target.value)}/>
+              
+              {/* <input
+              type="text"
+              value={state}
+              onChange={(event) => setState(event.target.value)} /> */}
+
+          <TextField
+            className={classes.inputs}
+            required
+            label='Country'
+            variant='filled'
+            value={country} 
+            type="text" 
+            onChange={(event) => setCountry(event.target.value)}/>
+              
+              {/* <input
+              type="text"
+              value={country}
+              onChange={(event) => setCountry(event.target.value)} /> */}
+        </div>
+        <div>
+
+          <TextField
+            className={classes.inputs}
+            required
+            label='Strain Of Crop'
+            variant='filled'
+            value={cropStrain} 
+            type="text" 
+            onChange={(event) => setCropStrain(event.target.value)}/>
+          
+          {/* <input
+            type="text"
+            value={cropStrain}
+            onChange={(event) => setCropStrain(event.target.value)} /> */}
+        </div>
+
+        <div>
+          <TextField
+            id="date"
+            label="Harvest Date"
+            type="date"
+            defaultValue="2021-01-01"
+            className={classes.inputs}
+            InputLabelProps={{
+              shrink: true,
+            }}/>
+        </div>
+        {/* <div
+          value={harvestDate}
+          onChange={(event) => setHarvestDate(event.target.value)}>
+          Harvest Date
+          <input
+            type="month" />
+        </div> */}
+
+        <div>
+          <TextField
+            className={classes.inputs}
+            label='Sustainability Info:'
+            variant='filled'
+            value={sustainability} 
+            type="text" 
+            onChange={(event) => setSustainability(event.target.value)}/>
+          
+        {/* <input
+            value={sustainability}
+            type="text"
+            onChange={(event) => setSustainability(event.target.value)} /> */}
+
+        </div>
+
+        <Button 
+        className={classes.inputs}
+        style={{ backgroundColor: "#1e565c", color: "white" }}
```

Comment:
Don't tell anyone, but I'm personally ok with using inline styles like that. 🤭 
The only "code smell" here is that duplication of those styles all over your code. That makes it a pain, if you ever want to tweak the styles.

Options:
- Have a `styles.js` file, that keeps track of those colors and such as variables
- Extract this Button into its own `MyAwesomeStyleOfButton` component, so your inline styles code is only in one spot.